src/main/java/ui/components/NavigableListView.java

```diff
@@ -111,25 +114,36 @@ public void restoreSelection() {
                 selectedIndex = Optional.of(nextIndex);
 
                 // The next item will be considered selected
-                onItemSelected.accept(nextIndex);
+                setItemSelectedParams(nextIndex, SHOULD_LOAD_BVIEW_ISSUE);
             }
         }
     }
 
+    /**
+     * Passes the relevant index and boolean value for whether the bview should be loaded
+     * to the callback variable onItemSelected
+     */
+    private void setItemSelectedParams(int index, boolean shouldLoadBViewIssue){
+        onItemSelected.accept(index, shouldLoadBViewIssue);
```

Comment:
I think the interface can be improved so this doesn't just delegate. How about something like `itemSelected(index)` and `itemSelectedSecondary(index)`? Exact names should be improved, turned into verbs, etc. but now we have two methods for dealing with the two cases, and it's clear from call sites how the selection callback is being invoked. We shouldn't need the constant with this.